apps/src/applab/designElements/dropdown.jsx

```diff
@@ -88,6 +88,16 @@ module.exports = {
     element.style.height = '30px';
     element.style.fontSize = '14px';
     element.style.margin = '0';
+    element.style.color = '#fff';
+    element.style.backgroundColor = '#1abc9c';
```

Comment:
Understood, but it'd be nice if we could have all our color-related css files in one directory (e.g. shared) at some point.  Also, if we need them in JS, can we just have the JS set a CSS class on the element rather than the actual RGB?